frontend/wasm/src/api.rs

```diff
@@ -552,16 +488,13 @@ impl JsEditorHandle {
 		self.dispatch(message);
 	}
 
-	/// Creates an empty folder at the document root
-	pub fn create_empty_folder(&self) {
-		let message = DocumentMessage::CreateEmptyFolder { container_path: vec![] };
+	// TODO: Replace with initialization system, issue #524
+	pub fn init_app(&self) {
+		// Switch away from, and back to, the Select Tool to make it display the correct hints in the status bar
+		let message = ToolMessage::ActivateTool { tool_type: ToolType::Path };
+		self.dispatch(message);
+		let message = ToolMessage::ActivateTool { tool_type: ToolType::Select };
```

Comment:
Can we not use `ToolMessage::UpdateHints` directly?

Reply:
Comment was out of date, the real issue is tool initialization. I've updated the comment like so:

> // Switch away from, and back to, the Select tool to make it initialize the active tool correctly
> // TODO: Fix the root cause of this issue to avoid this hacky workaround

If you have any ideas for a quick fix, that'd be great. Or we can leave it as is for now and deal with the TODO eventually.